src/client/project/__init__.py

```diff
@@ -62,49 +62,50 @@
 
 
 class Project(BaseTolokaObject):
-    """Top-level object in Toloka. All other entities are contained in some project.
+    """Top-level object in Toloka that describes one requester's task.
```

Comment:
I think, `task` is not exactly suitable word here, as we use tasks to denote the items we upload to the pool. Perhaps something like `goal` or `a set of tasks` will fit better here? Or maybe you have your own variant